### ISSUE TITLE
Okta requirement outdated

### ISSUE THREAD

**Issue by kennedy** [Nike-Inc/gimme-aws-creds#322]
## Expected Behavior

Run gimme-aws-creds with okta v2.X

## Current Behavior

Run gimme-aws-creds with okta in V2.2.0, latest of this writing.

Getting this error message:

```
File association not found for extension .py
Traceback (most recent call last):
  File "C:\Users\***\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_***\LocalCache\local-packages\Python39\Scripts\gimme-aws-creds.cmd", line 48, in <module>
    from gimme_aws_creds.main import GimmeAWSCreds
  File "C:\Users\***\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_***\LocalCache\local-packages\Python39\site-packages\gimme_aws_creds\main.py", line 24, in <module>
    from okta.framework.ApiClient import ApiClient
ModuleNotFoundError: No module named 'okta.framework'
```

## Steps to Reproduce (for bugs)

1. `pip install gimme-aws-creds`
2. `pip install --upgrade okta`
3. `gimme-aws-creds --profile awsaccount`

## Context

The required version of okta for gimme-aws-creds was a beta, and retired. The current supported version is 2.X (2.2.0)

### Okta Release status

https://github.com/okta/okta-sdk-python/blob/master/README.md?plain=1#L37-L47

| Version | Status                           |
| ------- | -------------------------------- |
| 0.x     | :warning: Beta Release (Retired) |
| 1.x     | :warning: Retired                |
| 2.x     | :heavy_check_mark: Release       |

## Your Environment

* App Version used: 2.4.3
* Environment name and version: Python 3.9.7
* Operating System and version: win10.19043.1288

**Comment by iam-decoder** [comment]
I have another program using the okta library and it requires 2.x

It's quite a pain to have to swap between each version of okta to have both programs run correctly.

**Comment by zelch** [comment]
Fixed in #366.

**Comment by epierce** [comment]
This was fixed in v2.8.0
